test/fixtures/amp-cupcake-ad.html

```diff
@@ -5,8 +5,8 @@
 /*# sourceURL=/extensions/amp-carousel/0.1/amp-carousel.css*/</style><meta content="width=device-width,minimum-scale=1,initial-scale=1" name="viewport"><meta content="vendor=doubleclick,type=impression-id,value=CM6Vy-Xl_N4CFcJgwQods48FUA" name="amp4ads-id"><meta content="noindex" name="robots"><link as="script" href="https://cdn.ampproject.org/amp4ads-v0.js" rel="preload"><script async="" src="https://cdn.ampproject.org/amp4ads-v0.js"></script><script async="" custom-element="amp-accordion" src="https://cdn.ampproject.org/v0/amp-accordion-0.1.js"></script><script async="" custom-element="amp-analytics" src="https://cdn.ampproject.org/v0/amp-analytics-0.1.js"></script><script async="" custom-element="amp-anim" src="https://cdn.ampproject.org/v0/amp-anim-0.1.js"></script><script async="" custom-element="amp-audio" src="https://cdn.ampproject.org/v0/amp-audio-0.1.js"></script><script async="" custom-element="amp-carousel" src="https://cdn.ampproject.org/v0/amp-carousel-0.1.js"></script><script async="" custom-element="amp-fit-text" src="https://cdn.ampproject.org/v0/amp-fit-text-0.1.js"></script><script async="" custom-element="amp-font" src="https://cdn.ampproject.org/v0/amp-font-0.1.js"></script><script async="" custom-element="amp-form" src="https://cdn.ampproject.org/v0/amp-form-0.1.js"></script><script async="" custom-element="amp-social-share" src="https://cdn.ampproject.org/v0/amp-social-share-0.1.js"></script><style amp-custom="">.body {}
 </style></head><body marginwidth="0" marginheight="0" class="amp-mode-mouse" style="opacity: 1; visibility: visible; animation: none 0s ease 0s 1 normal none running;"><amp-pixel allow-ssr-img="" class="i-amphtml-layout-fixed i-amphtml-layout-size-defined i-amphtml-element" i-amphtml-layout="fixed" i-amphtml-ssr="" src="http:__VIEW_URL__" style="width:1px;height:1px;" aria-hidden="true"><img src="http:__VIEW_URL__"></amp-pixel><a href="https://google.com/" target="_blank">
-    <amp-img class="i-amphtml-layout-fixed i-amphtml-layout-size-defined i-amphtml-element i-amphtml-layout" height="200" i-amphtml-layout="fixed" i-amphtml-ssr="" src="http://localhost:8081/test/fixtures/served/cupcake.jpg" style="width:300px;height:200px;" width="300">
-    <img class="i-amphtml-fill-content i-amphtml-replaced-content" decoding="async" src="http://localhost:8081/test/fixtures/served/cupcake.jpg"></amp-img>
+    <amp-img class="i-amphtml-layout-fixed i-amphtml-layout-size-defined i-amphtml-element i-amphtml-layout" height="200" i-amphtml-layout="fixed" i-amphtml-ssr="" src="http://localhost:__TEST_SERVER_PORT__/test/fixtures/served/cupcake.jpg" style="width:300px;height:200px;" width="300">
+    <img class="i-amphtml-fill-content i-amphtml-replaced-content" decoding="async" src="http://localhost:__TEST_SERVER_PORT__/test/fixtures/served/cupcake.jpg"></amp-img>
```

Comment:
Nice. How would you test that this works? Manually?

Reply:
The test on Travis passed, didn't it? :)